app/lib/irs1040_pdf.rb

```diff
@@ -1,92 +1,87 @@
 class Irs1040Pdf
-  attr_accessor :address
   include PdfHelper
 
   def source_pdf_name
     "f1040-TY2021"
   end
 
   def initialize(submission)
-    @submission = submission
-    @tax_return = submission.tax_return
-    @intake = submission.intake
-    @qualifying_dependents = submission.qualifying_dependents
-    @address = @submission.verified_address || @intake.address
-    @benefits = Efile::BenefitsEligibility.new(tax_return: @tax_return, dependents: @qualifying_dependents)
+    # For some PDF fields, use values from the database b/c the XML values are truncated or missing.
+    intake = submission.intake
+    @primary_first_name = intake.primary_middle_initial.present? ? "#{intake.primary_first_name} #{intake.primary_middle_initial}" : intake.primary_first_name
+    @primary_last_name = intake.primary_last_name
+    @spouse_first_name = intake.spouse_middle_initial.present? ? "#{intake.spouse_first_name} #{intake.spouse_middle_initial}" : intake.spouse_first_name
+    @spouse_last_name = intake.spouse_last_name
+    address = submission.verified_address || intake.address
+    @street_address = address.street_address
```

Comment:
to me, it would make sense to just call all of these methods on the address object when using them instead of instantiating the additional instance variables.

Reply:
Asheesh and I talked about it and felt like individually pulling out each specific entry that we take not from xml would help make it more clear that the default way we do it should be through xml